docs/rst/tutorials/cloud/secure_router/secure_router.rst

```diff
@@ -86,7 +86,30 @@ We can set up another domain or even a range of domains applying the following c
                 </id_range>
             </domains>
 
-For our specific case, we will set the domain to ``ROS_DOMAIN_ID=1`` in the *listener* and *Participant 1* permissions and a range including both 0 and 1 domains in the governance file.
+For our specific case, we will modify the corresponding *listener* and ROS 2 Router participant in domain 1 *permissions.xml* files to apply to ROS 2 Domain ``1``.
+In order to do so, please find the ``<domain></domain>`` section in
+
+* ``~/sros2_demo/demo_keystore/enclaves/ros2_router/participant1/permissions.xml``, and
+* ``~/sros2_demo/demo_keystore/enclaves/ros2_router/participant1/permissions.xml``
```

Comment:
I think one of this is `~/sros2_demo/demo_keystore/enclaves/talker_listener/listener/permissions.xml`